test/index.test.js

```diff
@@ -26,10 +26,13 @@ describe('fixtures', () => {
 		}
 
 		it(fixtureDir, async () => {
+			const prevDir = process.cwd();
+			process.chdir(path.resolve(fixturePath));
```

Comment:
without this neither `rollup-plugin-typescript` nor `rollup-plugin-typescript2` would find the `tsconfig.json` file in the fixture.